packages/core/contracts/financial-templates/common/FundingRatePayer.sol

```diff
@@ -58,7 +58,7 @@ abstract contract FundingRatePayer is FeePayer, PerpetualInterface {
 
     /**
      * @notice Sends `amount` fees to the FundingRateStore contract and debits the raw collateral accordingly.
-     * @dev Callable only by the FundingRateStore.
+     * @dev Callable only by the FundingRateStore. This should never revert.
```

Comment:
I made changes in this method so that it doesn't revert, we don't want the FundingRateStore to call this method and have it revert